pkg/test/resourcefixture/testdata/basic/spanner/v1beta1/spanneinstance-unset-compute-fields/create.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: spanner.cnrm.cloud.google.com/v1beta1
+kind: SpannerInstance
+metadata:
+  labels:
+    label-one: "value-one"
+  name: spannerinstance-sample-${uniqueId}
+spec:
+  config: regional-us-west1
+  displayName: Spanner Instance Sample
+  numNodes: 2
```

Comment:
Can we have both numNodes and processingUnit configured at the same time?

Reply:
No, API would give you an error when setting both fields.